.github/labeler.yml

```diff
@@ -0,0 +1,23 @@
+DB:
+  - src/**/*.sql
+
+CORE:
+  - src/**/*.cpp
+  - src/**/*.h
+  - '!src/server/scripts/*'
+
+Script:
+  - src/server/scripts/*.cpp
+  - src/server/scripts/*.h
```

Comment:
same remark as above

Reply:
actually you are right, it should be /**/*.cpp to include the subfolders too. My bad